akka-testkit-typed/src/test/scala/akka/testkit/typed/scaladsl/TestProbeSpec.scala

```diff
@@ -5,12 +5,14 @@
 package akka.testkit.typed.scaladsl
 
 import akka.actor.typed.scaladsl.Behaviors
-import org.scalatest.{ BeforeAndAfterAll, Matchers, WordSpecLike }
+import akka.testkit.typed.TestKitSettings
 
 import scala.concurrent.duration._
 
 class TestProbeSpec extends AbstractActorSpec {
 
+  override implicit def settings: TestKitSettings = testkitSettings
```

Comment:
TODO consider calling it `testKitSettings`

Reply:
Done, will update when everything has been reviewed.